US.pm

```diff
@@ -881,7 +893,7 @@ sub init {
             |stop
             |tra?i?le?r
             |pod
-            |p(?:ost)?\W*o(?:ffice)?\W*box
+            |(?:u\.?s)\W*p(?:ost)?\W*[0o](?:ffice)?\W*(?:box)?
```

Comment:
Would we want the `(?:u\.?s)` part here to be optional?

It looks like one test changes if we do that:

```
not ok 80 - can parse Post Office Box 12345
#   Failed test 'can parse Post Office Box 12345'
#   at t/01_parser.t line 1081.
#     Structures begin differing at:
#          $got->{sec_unit_type} = 'Post Office Box'
#     $expected->{sec_unit_type} = 'Office'
Got: $VAR1 = {
          'sec_unit_num' => '12345',
          'sec_unit_type' => 'Post Office Box'
        };
```

which seems like afterwards is possibly better.

Reply:
Yes, nice catch.